docs/src/pages/index.js

```diff
@@ -24,17 +23,45 @@ function Home() {
           </div>
         </header>
         <main>
-            <section className={styles.section}>
-              <h2>Contribution</h2>
-              <ul>
-                <li>Clone the repo from https://github.com/Calamity210/BirbLang</li>
-                <li>Navigate to the new directory and install birb by running:
-                  <div class="language-shell highlighter-rouge"><div class="highlight"><pre class="highlight language-shell"><code class=" language-shell">$ dart2native lib/birb.dart
-              </code></pre></div>    </div>
-                  </li>
-                <li>When ready, open and pull request and request a review from a contributor.</li>
-              </ul>
-            </section>
+            <div className={styles.section}>
+                <h1>Why Birb</h1>
+                <div>
+                  <h2>Easy to learn</h2>
+                  <p>Eaay to make apps in less amount of code and pretty easy to learn if you know any other language</p>
+                  <img src="img/code.svg" height="300px" color="#6ac00d"></img>
+                </div>
+                <div>
+                  <h2>Null Saftey</h2>
+                  <p>Avoid entire classes of errors such as null pointer exceptions</p>
```

Comment:
You wouldnt encounter a null pointer in dart nor in birb regardless, and null safety isnt really ready so I wouldn't put this up yet